grpc/src/test/java/com/linecorp/armeria/it/grpc/HttpJsonTranscodingTest.java

```diff
@@ -174,7 +175,7 @@ public void updateMessageV1(UpdateMessageRequestV1 request, StreamObserver<Messa
         @Override
         public void updateMessageV2(Message request, StreamObserver<Message> responseObserver) {
             final ServiceRequestContext ctx = ServiceRequestContext.current();
-            final String messageId = Optional.ofNullable(ctx.pathParam("message_id")).orElse("no_id");
+            final String messageId = Optional.ofNullable(ctx.pathParam("p0")).orElse("no_id");
```

Comment:
It seems like a breaking behavior.
What do you think of other approaches? e.g. prohibiting `p0`...`pn` as a parameter name or using different parameter names that a user wouldn't use(Still need a validation though).

Reply:
Good point. I've updated so that we prepend a '@' in front of generated path parameters. Let me know if this doesn't make sense.